Deliverable 0 & 1 - BU Sustainable Purchasing Program

milesway opened the pull request.

milesway committed:
Create ReadMe.md

milesway committed:
Update ReadMe.md

milesway committed:
Update ReadMe.md

milesway committed:
Update ReadMe.md

milesway committed:
Update ReadMe.md

gallettilance reviewed (changes requested):
the file should be named "README.md" (all caps)

milesway committed:
Rename ReadMe.md to README.md

XiongxinZ committed:
Give a general view of our data and remove some "Unclassified"

XiongxinZ committed:
Calculate the waste volume for top coffee categories

We found that the top 20 coffee products requested are K-Cup Pods and Coffee Creamers and then calculated the waste volume based on the online open source of data for K-Cup and Coffee Creamer.

Commit:
set up the structure for all deliverables

Commit:
Update details for Deliverable 1

XiongxinZ commented:
We need someone has write access to merge our previous pull requests in order to build a new pull @ankilinnn
